x-pack/plugins/infra/common/inventory_models/host/metrics/tsvb/host_network_traffic.ts

```diff
@@ -23,71 +23,35 @@ export const hostNetworkTraffic: TSVBMetricModelCreator = (
       id: 'tx',
       metrics: [
         {
-          field: 'system.network.out.bytes',
-          id: 'max-net-out',
-          type: 'max',
-        },
-        {
-          field: 'max-net-out',
-          id: 'deriv-max-net-out',
-          type: 'derivative',
-          unit: '1s',
-        },
-        {
-          id: 'posonly-deriv-max-net-out',
-          type: 'calculation',
-          variables: [{ id: 'var-rate', name: 'rate', field: 'deriv-max-net-out' }],
-          script: 'params.rate > 0.0 ? params.rate : 0.0',
-        },
-        {
-          function: 'sum',
-          id: 'seriesagg-sum',
-          type: 'series_agg',
```

Comment:
We can cut this because the new field already encapsulates this computation, right?

Reply:
Yes, though I've replaced it with some other "queries" needed for this data type